components/inputs/input-checkbox.js

```diff
@@ -181,7 +184,7 @@ class InputCheckbox extends InputInlineHelpMixin(FocusMixin(SkeletonMixin(LitEle
 				}
 				.d2l-input-checkbox-supporting {
 					display: none;
-					margin-block: 0.9rem;
+					margin-block-start: ${unsafeCSS(inputStyleTweaksEnabled ? '0.6rem' : '0.9rem')}; /* stylelint-disable-line */
```

Comment:
Switched this to only apply the margin above since the bottom margin of the "supporting" <div> and the bottom margin of the host control were merging (not combining) anyway.